modules/smartrtbBidAdapter.js

```diff
@@ -0,0 +1,105 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'smartrtb';
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['smrtb'],
+  isBidRequestValid: function(bid) {
+    return (bid.params.pubId !== null &&
+      bid.params.medId !== null &&
+      bid.params.zoneId !== null);
+  },
+  getDomain: function () {
```

Comment:
Instead of attaching this function onto the `spec` object, you could just make it an internal function.

This would be more ideal, as we generally prefer to have designated functions be attached to the `spec` object.

Reply:
Thank you for the input. I've moved getDomain outside of the exported spec.